### DISCUSSION TITLE
Education Benefits Not Recieved

### DISCUSSION THREAD

**Discussion by lindsaychijate** [community/community#136694] [GitHub Education]
### Hi everyone,

Hi, I was approved for the student developer pack on August 17, it is currently the 23 and I still have not gotten the educational benefits email. What should i do?

**Comment by pg56714** [comment]
If your application is approved, you will receive a confirmation email. Applications are generally processed within a few days, but during peak times, such as the start of a new semester, it may take longer.

If I am correct, please mark it as the answer. Thank you.
@lindsaychijate

**Comment by davevad93** [comment]
Hi @lindsaychijate ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @lindsaychijate was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
